authbridge/storage/redis/redis_test.go

```
@@ -0,0 +1,186 @@
+
+	"github.com/alicebob/miniredis/v2"
+)
+
+func setup(t *testing.T) (*Client, *miniredis.Miniredis) {
+	t.Helper()
+	mr := miniredis.RunT(t)
+	c, err := New("redis://" + mr.Addr())
+	if err != nil {
+		t.Fatalf("New: %v", err)
+	}
+	t.Cleanup(func() { c.Close() })
```

golangci-lint (2.12.2)

[error] Error return value of `c.Close` is not checked

(errcheck)